gtfu/src/main/java/gtfu/GPSStats.java

```diff
@@ -0,0 +1,48 @@
+package gtfu;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import java.math.RoundingMode;
+import java.text.DecimalFormat;
+
+public class GPSStats {
```

Comment:
Nothing about this is specific to GPS. Can it just be a stats class?

Reply:
Yes - though I'm not sure how to accept a generic object class as input rather than GPSData. Updating to Stats for now.